Resfeber/Model/Destination.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Destination.swift
+//  Resfeber
+//
+//  Created by Joshua Rutkowski on 11/7/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+class Destination {
```

Comment:
If you ultimately want to be able to encode/decode Destination you'll want to make sure it conforms to codable. Just FYI though, UIImage is not codable in it of itself so you'll have to work around that, probably using `imageData`